playbooks/installation/installation.adoc

```diff
@@ -285,35 +113,22 @@ Overall requirements for Installing OpenShift are very simple:
 ** rhel-7-server-ose-3.0-rpms
```

Comment:
rhel-7-server-optional-rpms (not required - so should be removed?)

rhel-7-server-ose-3.0-rpms (update to 3.0 to 3.2 or 3.x)  could add note for versions like  `Note: x = minor version`